iengine/iengine-app/src/main/java/io/tapdata/flow/engine/V2/node/hazelcast/HazelcastBaseNode.java

```diff
@@ -561,11 +565,36 @@ public synchronized TapCodeException errorHandle(Throwable throwable, String err
 		TapCodeException currentEx = wrapTapCodeException(throwable);
 		TaskDto taskDto = processorBaseContext.getTaskDto();
 		handleWhenTestRun(taskDto, currentEx);
-
+		List<ErrorEvent> errorEvents;
+		if(CollectionUtils.isNotEmpty(taskDto.getErrorEvents())){
+			errorEvents = taskDto.getErrorEvents();
+		}else{
+			errorEvents = new ArrayList<>();
+		}
+
+		List<ErrorEvent> skipErrorEvents = errorEvents.stream().filter(ErrorEvent::getSkip).collect(Collectors.toList());
+		SkipError skipError = SkipErrorStrategy.getSkipErrorStrategy(null).getSkipError();
+		if(CollectionUtils.isNotEmpty(skipErrorEvents)){
+			if(skipError.match(skipErrorEvents,new ErrorEvent(errorMessage, Log4jUtil.getStackStrings(currentEx)))){
+				obsLogger.warn(errorMessage + "; Will skip it");
```

Comment:
这里改为info级别，并用固定的前缀方便以后排查
```suggestion
				obsLogger.info("The current exception match the skip exception strategy"+errorMessage);
```